lib/HttpStatus.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace Aerys;
+
+/**
+ * @link https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml
+ */
+final class HttpStatus {
```

Comment:
This could be useful in amp/artax as well, maybe it would be nice to have an amp/http-message library in the future containing this code.

Reply:
Moved to `amphp/http` now.